templates/al2/runtime/bootstrap.sh

```diff
@@ -465,13 +465,17 @@ if [[ -z "${DNS_CLUSTER_IP}" ]]; then
 
   if [[ "${IP_FAMILY}" == "ipv4" ]]; then
     if [[ ! -z "${SERVICE_IPV4_CIDR}" ]] && [[ "${SERVICE_IPV4_CIDR}" != "None" ]]; then
-      #Sets the DNS Cluster IP address that would be chosen from the serviceIpv4Cidr. (x.y.z.10)
+      # Sets the DNS Cluster IP address that would be chosen from the serviceIpv4Cidr. (x.y.z.10)
+      log "INFO: Detected SERVICE_IPV4_CIDR='$SERVICE_IPV4_CIDR'. Will set DNS_CLUSTER_IP as '${SERVICE_IPV4_CIDR%.*}.10'."
       DNS_CLUSTER_IP=${SERVICE_IPV4_CIDR%.*}.10
     else
       TEN_RANGE=$(imds "latest/meta-data/network/interfaces/macs/$MAC/vpc-ipv4-cidr-blocks" | grep -c '^10\..*' || true)
       DNS_CLUSTER_IP=10.100.0.10
       if [[ "$TEN_RANGE" != "0" ]]; then
+        log "INFO: Detected no SERVICE_IPV4_CIDR defined and VPC CIDR (IPv4) in 10.0.0.0/8 range. Will set DNS_CLUSTER_IP as '172.20.0.10'."
         DNS_CLUSTER_IP=172.20.0.10
+      else
+        log "INFO: Detected no SERVICE_IPV4_CIDR defined and VPC CIDR (IPv4) not in 10.0.0.0/8 range. Will set DNS_CLUSTER_IP as '10.100.0.10'."
```

Comment:
Even though it is detected automatically, it should at least leave some message to debug purpose.